api/src/authz/intents.ts

```diff
@@ -68,6 +71,9 @@ export const globalIntents: Intent[] = [
   "global.createGroup",
   "user.authenticate",
   "user.changePassword",
+  "user.intent.listPermissions",
+  "user.intent.grantPermission",
+  "user.intent.revokePermission",
```

Comment:
as discussed please remove this and the `user.changePassword` intent from the list of global intents

Reply:
remoed